test/test_backend/routes/tst_graph.py

```diff
@@ -45,57 +45,59 @@
 TEST_CASES = TEST_CASES_FAST
 EXTRA_CONCEPT_IDS = []
 HIDE_VOCABS = ['RxNorm Extension']
-CONNECTED_SUBGRAPH_FILENAME_PATTERN = 'get_connected_subgraph__{}.json'
 
 
 # class TestGraph(unittest.TestCase):
 class TstGraph:
     """Tests for graph.py"""
     test_cases: List[Dict] = []
 
-    # noinspection PyPep8Naming
-    @classmethod
-    def setUpClass(cls):
+    @staticmethod
+    def _get_test_cases() -> List[Dict]:
         """Set up"""
         csv_file = StringIO(TEST_CASES_FAST.strip())
         cases: List[Dict] = list(csv.DictReader(csv_file, delimiter=',', quotechar='"'))
         for case in cases:
             for k, v in case.items():
                 if k == 'codeset_ids':
                     case[k] = v.split(',')
-        cls.test_cases = cases
+        return cases
+
+    # TODO: type hints
+    @staticmethod
+    def _save_output(
```

Comment:
## Saves output
Now by default saves outputs when test is run to a subfolder which is named for the given git commit hash.

> [!Warning]
> Make sure all changes are committed before running and saving outputs. Otherwise the hash will be incorrect.